src/3d/qgs3dmapscene.cpp

```diff
@@ -1045,7 +1045,12 @@ void Qgs3DMapScene::exportScene( const Qgs3DMapExportSettings &exportSettings )
   if ( mTerrain )
     exporter.parseTerrain( mTerrain, "Terrain" );
 
-  exporter.save( exportSettings.sceneName(), exportSettings.sceneFolderPath() );
+  const bool sceneSaved = exporter.save( exportSettings.sceneName(), exportSettings.sceneFolderPath() );
+  if ( !sceneSaved )
+  {
+    QgsMessageOutput::showMessage( tr( "3D exporter warning" ), tr( "The scene can not be exported." ), QgsMessageOutput::MessageText );
```

Comment:
Can we use a messagebar message here instead? (It'd be nice to show a "success" message too, just like we do eg here: https://github.com/qgis/QGIS/blob/master/src/app/layout/qgslayoutdesignerdialog.cpp#L2268 )

Reply:
Done.